ext/ftp/tests/ftp_ssl_connect_error.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Test ftp_ssl_connect() function : error conditions
+--SKIPIF--
+<?php
+$ssl = 1;
+require 'skipif.inc';
+if (!function_exists("ftp_ssl_connect")) die("skip ftp_ssl is disabled");
+?>
+--FILE--
+<?php
+echo "*** Testing ftp_ssl_connect() function : error conditions ***\n";
+echo "\n-- Testing ftp_ssl_connect() function on failure --\n";
+var_dump(ftp_ssl_connect('127.0.0.1'));
+
+echo "\n-- Testing ftp_ssl_connect() function timeout warning for value 0 --\n";
+ftp_ssl_connect('127.0.0.1', 21, 0);
+
+?>
```

Comment:
@chancegarcia s|ll means:

s - parameter #1 must be a string
| - following parameters are optional
l - parameter #2 must be a numeric (integer)
l - parameter #3 must be a numeric (integer)

To make this fail, you can pass something like
new stdClass
[]

To the first parameter, and the conditional should fail. If you look in the ext/standard/tests directory, there should be a lot of error condition tests you can use for reference